document/core/exec/instructions.rst

```diff
@@ -534,7 +534,7 @@ SIMD instructions are defined in terms of generic numeric operators applied lane
 
 5. Let :math:`j^\ast` be the sequence :math:`\lanes_{t\K{x}N}(c_2)`.
 
-6. Let :math:`c` be the result of computing :math:`\lanes^{-1}_{t\K{x}N}(\extend^s_{i1,t}(\vrelop_t(i^\ast, j^\ast)))`.
+6. Let :math:`c` be the result of computing :math:`\lanes^{-1}_{t\K{x}N}(\extend^s_{\i1,t}(\vrelop_t(i^\ast, j^\ast)))`.
```

Comment:
Nit: there are \extendu and \extends macros that typeset the s/u correctly.

I'd still suggest to got to the bit-width version directly. The "automatic conversion" is just intended as a way to define the actual instructions of that name. Here it's just an unnecessary step. (Accordingly, the aforementioned macros point to the base definition.)

Reply:
Oh yes I forgot about those. Fixed according to your suggestion, and removed \i1 since it's not longer used anywhere, thanks!